src/index.html

```diff
@@ -15,8 +15,86 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          class="logo__image"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav__link">
+        <ul class="ul__link">
+          <a
+            class="is-active link"
+            href="#"
+          >
+            <li class="list__item">Apple</li>
+          </a>
```

Comment:
These tags should be swapped, fix it everywhere
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">Apple</a>
          </li>
```